code/modules/ntsl2/telecomms.dm

```diff
@@ -0,0 +1,48 @@
+/datum/TCS_Compiler/ntsl2
+	var/datum/ntsl_program/running_code = null
+
+	Compile(code as message)
```

Comment:
`as message` is useless in the context of a proc. Unless used as a verb, but I don't think this is.

Reply:
Borrowed from the NTSL compiler, I'll change it.